### PR TITLE
STOC17/19/20 feature/setup CORS, added Config folder, and updated data mapping

### PR DESCRIPTION
**Pull request by Roboto0**

### CODE REVIEW

File: stockAndPrices/services/app/main.py [hunk 1]
```diff
@@ -14,10 +19,13 @@ def login():
     if request.method == "GET":
         return "<h1>Login Page</h1>"
     else:
-        user = request.form['user_name']
+        user = request.form.get('username')
+        #pwd = request.form.get('password')
```
Comment (allenaerostar) [inline stockAndPrices/services/app/main.py:23]: Lets removed these commented out code.

File: stockAndPrices/services/app/main.py [hunk 2]
```diff
@@ -14,10 +19,13 @@ def login():
...
+        #print(user)
```
Comment (allenaerostar) [inline stockAndPrices/services/app/main.py:24]: Let us remove the commented out print statements, the password field we can leave as we will use it later

File: stockAndPrices/services/app/main.py [hunk 3]
```diff
@@ -1,7 +1,12 @@
 from flask import Flask, request
+from flask_cors import CORS
+from configuration.Origin import Origin
 
 
 app = Flask(__name__)
+app.config.from_pyfile("configuration/DevConfig.py")
+
+cors = CORS(app, resources={r"/.*": {"origins": Origin}})
```
Comment (allenaerostar) [inline stockAndPrices/services/app/main.py:9]: are we allowing all origins to hit, or just the react app origin here?

File: stockAndPrices/services/app/main.py (hunk 2 above)
Comment (Roboto0) [reply stockAndPrices/services/app/main.py]: Removed

File: stockAndPrices/services/app/main.py (hunk 3 above)
Comment (Roboto0) [reply stockAndPrices/services/app/main.py:9]: Yes, "Origin" was imported from Origin.py in the configuration folder,

**Review by allenaerostar — APPROVED** [review]

File: stockAndPrices/services/app/main.py [hunk 4]
```diff
@@ -1,7 +1,12 @@
 from flask import Flask, request
+from flask_cors import CORS
+from configuration.Origin import Origin
 
 
 app = Flask(__name__)
+app.config.from_pyfile("configuration/DevConfig.py")
```
Comment (allenaerostar) [inline stockAndPrices/services/app/main.py:7]: Can you change this to configuration/Config.py?

Comment (Roboto0) [reply stockAndPrices/services/app/main.py]: Updated

**Review by lilianahechen — APPROVED** [review]

**Review by allenaerostar — APPROVED** [review]

**Pull request merged**